app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecordRepository.kt

```diff
@@ -0,0 +1,49 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord
+
+import org.simple.clinic.storage.Timestamps
+import org.simple.clinic.util.UtcClock
+import org.simple.clinic.uuid.UuidGenerator
+import java.time.Instant
+import java.util.UUID
+import javax.inject.Inject
+
+class TeleconsultRecordRepository @Inject constructor(
+    private val uuidGenerator: UuidGenerator,
```

Comment:
I am wondering if we should move this dependency to the caller of the methods (i.e., the effect handlers) and just have the methods in the repository ask for the UUID to use directly as a parameter.

Reply:
Yeah, that works. Should I do it in this PR or something we need to look into further?

Reply:
We can move it later, not an issue. But moving it now would simplify the code since we won't have to make a default parameter override for the purpose of setting the UUID in tests.